app/components/Feed/renders/restrictedMail.tsx

```diff
@@ -8,7 +8,7 @@ import type { AggregatedActivity } from '../types';
  * No extra information in the feed element
  */
 export function activityHeader() {
-  return <b>Begrenset epost sent ut til alle mottakere</b>;
+  return <b>Begrenset e-post sent ut til alle mottakere</b>;
```

Comment:
```suggestion
  return <b>Begrenset e-post sendt ut til alle mottakere</b>;
```

Reply:
Nicely spotted. I just did a quick replace all `epost` -> `e-post` so I didn't read through it all, but I'll definitely note it!